qa/L0_backend_python/python_test.py

```diff
@@ -263,6 +315,14 @@ def test_unicode(self):
                     self.assertIsNotNone(output0)
                     self.assertEqual(output0[0], input_data)
 
+    def test_optional_input(self):
+        model_name = "optional"
+
+        with self._shm_leak_detector.Probe() as shm_probe:
+            for input0 in [True, False]:
```

Comment:
input0/input1 -> skip_input0/skip_input1

The variable name should clarify that it is boolean. Note: there will be reversal in logic computation.